src/container/container_iv.c

```diff
@@ -479,8 +479,9 @@ cont_iv_ent_fetch(struct ds_iv_entry *entry, struct ds_iv_key *key,
 				rc = cont_iv_prop_ent_create(entry, key);
 				if (rc == 0)
 					goto again;
-				D_ERROR("create cont prop iv entry failed "
-					""DF_RC"\n", DP_RC(rc));
+				D_ERROR("cont " DF_UUID " create IV_CONT_PROP iv entry failed "
```

Comment:
suggest use DL_ERROR(rc, ...) here

also it may be helpful to log the container UUID in the other log calls in this function like D_DEBUG above on line 476 for IV_CONT_SNAP case and the D_DEBUG on line 546 at the end of the function.

Reply:
sure, this PR included in https://github.com/daos-stack/daos/pull/15994